components/raftstore-v2/src/operation/query/local.rs

```diff
@@ -50,6 +49,78 @@ where
     }
 }
 
+pub type ReadDelegatePair<EK> = (ReadDelegate, SharedReadTablet<EK>);
+
+/// A share struct for local reader.
+///
+/// Though it looks like `CachedTablet`, but it only stores one version of
```

Comment:
Not easy to understand. Maybe something like "Allow weak referencing a tablet. The initial instance of `SharedReadTablet` controls the lifetime of tablet. Subsequent clones can only temporarily borrow it if the initial instance is still alive".